cloud/services/availabilityzones/availabilityzones_test.go

```diff
@@ -97,7 +114,7 @@ func TestGetAvailabilityZones(t *testing.T) {
 				Cluster: cluster,
 				AzureCluster: &infrav1.AzureCluster{
 					Spec: infrav1.AzureClusterSpec{
-						Location: "test-location",
+						Location:      "centralus",
```

Comment:
extra indent

Reply:
Tried to change this but `go fmt` but the change back